docs/readmes/architecture/agw_overview.md

```diff
@@ -0,0 +1,60 @@
+* Device Data model : TR-181 and TR-098
+* Information Data model : TR-196
+
+4. **Magmad** - Parent service to start all Magma services, owns the collection and reporting of metrics of services, and also acts as the bootstrapping client with Orchestrator.
+
+5. **Dnsd** - local DNS and DHCP server for the eNB. 
+
+6. **Subscriberdb** - Magma uses Subscriberdb to enable LTE data services through one network node like AGW for LTE subscribers. It is deactivated for the deployments that make use of the MNO's HSS. It supports the following two S6a procedures: S6a: Authentication Information Request and Answer (AIR/AIA) and S6a: Update Location Request and Answer (ULR/ULA). Subscriberdb also supports these additional functions:
+   * Interface with Orchestrator to receive subscriber information such as IMSI, secret key (K), OP, user-profile during system bring-up.
+   * Generate Authentication vectors using* *Milenage Algorithm and share these with MME.
+   * Share user profile with MME.
+
+7. **Mobilityd** - IP Address Management Service. It primarily functions as an interface with the orchestrator to receive an IP address block during system bring-up. It can also allocate and release IP addresses for the subscriber on the request from S-PGW Control Plane.
+
+8. **Directoryd** - Lookup service where you are able to push different keys and attribute pairs for each key
+
+9. **Sessiond** - Sessiond implements the control plane for the PCEF functionality in Magma. Sessiond is responsible for the lifecycle management of the session state (credit and rules) associated with a user. It interacts with the PCEF datapath through pipelined for L2-L4 and DPId for L4-L7 policies.
+
+10. **Policydb** - PolicyDB is the service that supports static PCRF rules. This service runs in both the AGW and the orchestrator. Rules managed through the rest API are streamed to the policydb instances on the AGW. Sessiond ensures these policies are implemented as specified.
+
+11. **Dpid** - Deep packet inspection service to enforce policy rules.
+
+12. **Pipelined** - Pipelined is the control application that programs the OVS openflow rules. Pipelined is a set of services that are chained together. These services can be chained and enabled/disabled through the REST API. If pipelined is restarted, users service will be affected.
+
+13. **Eventd** - Service that acts like an intermediary for different magma services, using the service303 interface, it will receive and push the generated registered events to the td-agent-bit service on the gateway, so these can be then later sent to Orchestrator. These events will be sent to ElasticSearch where they can be queried. 
+
+14. **Monitord ** - a dynamic service that monitors the CPEs connected to the AGW. It will send ICMP pings to the CPEs connected to the gateway and report if they are active. 
```

Comment:
```suggestion
14. **Monitord** - a dynamic service that monitors the CPEs connected to the AGW. It will send ICMP pings to the CPEs connected to the gateway and report if they are active. 
```